tensorflow/contrib/cmake/tf_python.cmake

```diff
@@ -172,6 +172,7 @@ add_python_module("tensorflow/core/lib")
 add_python_module("tensorflow/core/lib/core")
 add_python_module("tensorflow/core/protobuf")
 add_python_module("tensorflow/core/util")
+add_python_module("tensorflow/examples/tutorials/mnist")
```

Comment:
I'm not sure, but I think it might be necessary to add the following lines as well:

```
add_python_module("tensorflow/examples")
add_python_module("tensorflow/examples/tutorials")
```

...or else the appropriate `__init__.py` files won't be touched. Can you check if that's the case?